src/java/arjdbc/postgresql/PGDriverImplementation.java

```diff
@@ -164,6 +166,28 @@ private static ByteList formatInterval(final Object object) {
         return str;
     }
 
+    private static Class<?> pgArrayClass;
+    private static Class<?> pgArrayClass(final ThreadContext context) {
+        final Class<?> clazz = pgArrayClass;
+        if (clazz == null) {
+            final Ruby runtime = context.getRuntime();
+            try {
+                // try loading official PgArray Class (since postgresql 9.4)
+                pgArrayClass = runtime.getJavaSupport().loadJavaClass("org.postgresql.jdbc.PgArray");
+                return pgArrayClass;
+            }
+            catch (ClassNotFoundException ex) {
+                try {
+                    // load older (internal) Jdbc4Array Class (9.2 and earlier)
+                    pgArrayClass = runtime.getJavaSupport().loadJavaClass("org.postgresql.jdbc4.Jdbc4Array");
```

Comment:
same here ... although I am not sure if it matters - it probably will find the class assuming parent delegation is on